web/packages/teleport/src/services/integrations/types.ts

```diff
@@ -632,6 +633,20 @@ export type AWSOIDCDeployedDatabaseService = {
   matchingLabels: Label[];
 };
 
+// AwsRolesAnywherePingResponse contains the result of the Ping request.
+// This response contains meta information about the current state of the Integration.
+export type AwsRolesAnywherePingResponse = {
+  // profileCount is the number of IAM Roles Anywhere Profiles that can be accessed by the Integration.
+  // Profiles that are disabled or don't have any IAM Role associated with them are not counted.
+  profileCount: number;
+  // accountID number of the account that owns or contains the calling entity.
+  accountID: string;
+  // arn associated with the calling entity.
+  arn: string;
+  // userID is the unique identifier of the calling entity.
+  userId: string;
+};
```

Comment:
can we convert the `//` comment to the `/***/` style? (it's what we prefer now and i think it's b/c it plays nicer with on hover show comments?)